pennylane/templates/parameters.py

```diff
@@ -0,0 +1,225 @@
+# Copyright 2018 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Parameters
+==========
```

Comment:
Something that I couldn't debug when compiling with Sphinx - I get an error about `parameters.py` not having a title?

Reply:
By the way, did you manage to solve this one? Really drove me crazy for half an hour the other day :laughing:

Reply:
I am about to go mad. No idea. I does have a title! Argh.

Reply:
It works for any other title than "Parameters", but for that particular name we get the "no title" error. Very strange.

Reply:
Simply renamed the title

Reply:
How bizarre!! That was really driving me mad